### DISCUSSION TITLE
Broken SSL certificate?

### DISCUSSION THREAD

**Discussion by St3V0Bay** [Project-MONAI/MONAILabel#556] [Q&A] [answered]
Dear MONAILabel team,

I had two instances of an SSL certificate error. The first one while pip installing monailabel, which could be "solved" by using the following command.
`pip install monailabel --trusted-host pypi.org --trusted-host files.pythonhosted.org`

At the next one I am stuck now. After successfully downloading app and dataset, I try to start the server. At the point where it tries to download the pretrained model is crashes. Please check the screenshot for more details.

Could you check whether you can reproduce it? If not, then the issue might be on my end... Any pointers would help.

![Screenshot 2021-12-10 at 14 29 51](https://user-images.githubusercontent.com/87860698/145581988-e61393b1-ac2b-40e6-853a-baa943e7af18.png)

**Comment by SachidanandAlle** [comment]
Are u trying to run monailabel under https?

https://docs.monai.io/projects/label/en/latest/installation.html#deployment

https://www.uvicorn.org/#command-line-options

**Reply by SachidanandAlle** [reply]
Based on pip error..  I can see that your node/box is trying fetch everything over ssl.. basically converting all http requests to https..

**Reply by St3V0Bay** [reply]
Thanks for the quick answer. I just wanted to run the vanilla example for the monai server without any changes. Do I understand correctly, that actually the server should start under http?

**Comment by SachidanandAlle** [comment]
Yes.. default is http..

**Reply by St3V0Bay** [reply]
so it seems, that some OS settings on my machine redirect any http request to https. Is that possible? Because, I did not explicitly chose https. I don't even know how to do that

**Comment by St3V0Bay** [comment, marked as answer]
Hi @SachidanandAlle,

The issue really seems to be caused by some settings on my machine, which are imposed by the IT Dept of my company. I could avoid certain errors by manually downloading a couple of needed artifacts and putting them in the right place.
